openreview/venue/matching.py

```diff
@@ -1165,6 +1165,23 @@ def deploy_assignments(self, assignment_title, overwrite):
             reviewer_name = venue.area_chairs_name
             review_name = 'Meta_Review'
 
+        # Remove reviewers_proposed_assignment_title if deploying reviewer assignments
```

Comment:
I would add this change after we confirm the assignments are deployed, this means after the line `openreview.tools.post_bulk_edges(client=client, edges=assignment_edges)`